tox.ini

```diff
@@ -78,6 +78,24 @@ commands =
         lib \
         ctl
 
+[testenv:check-py37]
+deps =
+    -r{toxinidir}/requirements.txt
+    pytest-cov==2.6.1
+    pytest==4.1.0
+basepython = python3.7
+changedir = {toxinidir}/tests
+commands =
+    pytest \
+        --log-level=DEBUG \
+        --durations=5 \
+        --cov=libnmstate \
+        --cov=nmstatectl \
+        --cov-report=html:htmlcov-py37 \
```

Comment:
Please also add `--cov-report=term \`

Reply:
my previous comment is not yet addressed.

Reply:
Done.